src/Spago/Registry.purs

```diff
@@ -62,6 +63,7 @@ type RegistryEnv a = Record (RegistryEnvRow a)
 type RegistryFunctions =
   { getManifestFromIndex :: PackageName -> Version -> Spago (LogEnv ()) (Maybe Manifest)
   , getMetadata :: PackageName -> Spago (LogEnv ()) (Either String Metadata)
+  , getMetadatas :: Array PackageName -> Spago (LogEnv ()) (Either String (Map PackageName Metadata))
```

Comment:
Is there any point in keeping `getMetadata` as well now that we have `getMetadatas`?

As an aside, `data` is a latin word that's already plural, so adding the `s` suffix doesn't feel right. How about renaming to `getMetadataForPackages`?

Reply:
The singular version is still used in Command.Repl, Command.Publish and Command.Registry so I just left it alone.

Thanks, will swap to suggested name, had the same thought re datas but wasn't sure about a good replacement.

Reply:
> The singular version is still used in Command.Repl, Command.Publish and Command.Registry so I just left it alone.

Sure, but we can now call the new implementation passing in a singleton array `[packageName]`? Having two implementations for the same thing guarantees that they will go out of sync in the future, so I'd rather remove the singular version now.

Reply:
Oh sure. Like this?

Reply:
I could replace the function entirely but it feels useful being able to look up a single package without `Map.lookup`ing the results.

Reply:
Yeah that makes sense, thanks!